pacu.py

```diff
@@ -1,1725 +1,3 @@
-
-        if export.lower() == 'y':
-            session = self.get_active_session()
-
-            if not session.access_key_id:
-                print('  No access key has been set. Not exporting credentials.')
-                return
-            if not session.secret_access_key:
-                print('  No secret key has been set. Not exporting credentials.')
-                return
-
-            config = """
-\n\n[{}]
-aws_access_key_id = {}
-aws_secret_access_key = {}
-""".format(session.key_alias, session.access_key_id, session.secret_access_key)
-            if session.session_token:
-                config = config + 'aws_session_token = "{}"'.format(session.session_token)
-
-            config = config + '\n'
-
-            with open('{}/.aws/credentials'.format(os.path.expanduser('~')), 'a+') as f:
-                f.write(config)
-
-            print('Successfully exported {}. Use it with the AWS CLI like this: aws ec2 describe instances --profile {}'.format(
-                session.key_alias, session.key_alias
-            ))
-        else:
-            return
-
-    # ***** Some module notes *****
-    # For any argument that needs a value and a region for that value, use the form
-    # value@region
-    # Arguments that accept multiple values should be comma separated.
-    #
-    def exec_module(self, command: List[str]) -> None:
-        session = self.get_active_session()
-
-        # Run key checks so that if no keys have been set, Pacu doesn't default to
-        # the AWSCLI default profile:
-        if not session.access_key_id:
-            print('  No access key has been set. Not running module.')
-            return
-        if not session.secret_access_key:
-            print('  No secret key has been set. Not running module.')
-            return
-
-        module_name = command[1].lower()
-        module = import_module_by_name(module_name, include=['main', 'module_info', 'summary'])
-
-        if module is not None:
-            # Plaintext Command Log
-            self.print('{} ({}): {}'.format(session.access_key_id, time.strftime("%a, %d %b %Y %H:%M:%S", time.gmtime()), ' '.join(command).strip()),
-                       output='file', is_cmd=True)
-
-            # TODO: XML Command Log - Figure out how to auto convert to XML
-            # self.print('<command>{}</command>'.format(cmd), output_type='xml', output='file')
-
-            self.print('  Running module {}...'.format(module_name))
-
-            try:
-                args = module.parser.parse_args(command[2:])
-                if 'regions' in args and args.regions is None:
-                    session = self.get_active_session()
-                    if session.session_regions == ['all']:
-                        if not self.all_region_prompt():
-                            return
-            except SystemExit:
-                print('  Error: Invalid Arguments')
-                return
```

Comment:
This logic was removed in main.py. This causes the following command to exit the RPEL.

```
run ebs__explore_snapshots --instance-id i-1234 --zone us-east-1
```

I don't think what's happening here really makes sense in general but doesn't make sense to change this behavior as apart of this change.